save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/AvatarStorage.kt

```diff
@@ -51,32 +54,37 @@ class AvatarStorage(configProperties: ConfigProperties) :
      * @return [Path] is built by [AvatarKey] object
      */
     override fun buildPathToContent(rootDir: Path, key: AvatarKey): Path = rootDir
-        .let { if (key.type == AvatarType.USER) it.resolve(USERS_DIRECTORY) else it }
+        .let {
```

Comment:
For the sake of readability, this is better replaced with

```kotlin
run {
    // ...
    rootDir.resolve(/* ... */)
    // ...
}
```

Second, `resolve` can be replaced with `/` (`operator fun div`).

Up to you, though.